Day4and5/src/components/SearchComponent/Search.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+
+import * as classes from "./Search.styles.module.css";
+import Data from "../../pages/HomePage/AvengersData.json";
+import { filterValues } from "../../utils";
+
+const search = ({ avengersData, setAvengersData }) => {
+  // State to preserve the input entered when enter is pressed
+  const [searchValue, setSearchValue] = useState("");
+
+  /**
+   * Handler for setting searchValue state
+   */
+  const handleChange = (e) => {
+    setSearchValue(e.target.value);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    // If no input provided then display all the data
+    if ((e.target.value = "")) {
+      setAvengersData(Data);
+      return;
```

Comment:
instead of return, better to have else statement. While writing test cases in future it will be easier to write.

Reply:
Yes, will change it.